test-resources/integration-tests/start/start-happy.test.ts

```diff
@@ -1,51 +1,54 @@
 import { stackOutputs } from "../helpers/cloudformation";
-import { signedFetch } from "../helpers/fetch";
 import { kmsClient } from "../helpers/kms";
 import { JweDecrypter } from "../helpers/jwe-decrypter";
 import { JwtVerifierFactory, ClaimNames } from "../helpers/jwt-verifier";
 import { getParametersValues } from "../../headless-core-stub/utils/src/parameter/get-parameters";
-
+import { base64Encode } from "../../headless-core-stub/utils/src/base64/index";
+import { Logger } from "@aws-lambda-powertools/logger";
+import { signedFetch } from "../helpers/fetch";
 describe("happy path core stub start endpoint", () => {
-    let authenticationAlg;
-    let publicSigningJwkBase64;
-    let testHarnessExecuteUrl;
-    let jweDecrypter;
+    let authenticationAlg: string;
+    let testHarnessExecuteUrl: string;
+    let jweDecrypter: JweDecrypter;
 
-    const jwtVerifierFactory = new JwtVerifierFactory();
+    const jwtVerifierFactory = new JwtVerifierFactory(new Logger());
     const clientId = "ipv-core-stub-aws-headless";
     const aud = "https://test-aud";
-    const iss = "https://test-issuer";
+    let iss: string;
 
     beforeAll(async () => {
         const { TestHarnessExecuteUrl, CommonStackName } = await stackOutputs(process.env.STACK_NAME);
         testHarnessExecuteUrl = TestHarnessExecuteUrl;
+        iss = TestHarnessExecuteUrl.replace(/\/+$/, "");
```

Comment:
Why do we have to do this?

Reply:
It to remove `https://test-resources.review-hc.dev.account.gov.uk/`

Reply:
Hmm what? Looks like it removes the trailing slash, is that what you mean? 
![image](https://github.com/user-attachments/assets/eeb95b11-6e7d-452e-ab76-c5a3b80aaaa4)

Reply:
Ok so if we have to do manipulation is stack outputs the right thing to be using for this? Should we just use the SSM param?